tests/src/JIT/Directed/Misc/gettype/gettypetypeofmatrix.csproj

```diff
@@ -22,8 +22,7 @@
     </CodeAnalysisDependentAssemblyPaths>
   </ItemGroup>
   <PropertyGroup>
-    <!-- Set to 'Full' if the Debug? column is marked in the spreadsheet. Leave blank otherwise. -->
-    <DebugType></DebugType>
+    <DebugType>None</DebugType>
```

Comment:
If you do not care much for the debug information at the moment, I'd just pass "None" for now. 

It is likely your CSC is missing this fix:
https://github.com/dotnet/roslyn/commit/bbe2d961c1cf17540dbd875a97e079ca83c04233